pf9/cluster/exceptions.py

```diff
@@ -51,8 +52,10 @@ def __init__(self, msg):
 
 
 class PrepNodeFailed(ClusterCLIException):
-    def __init__(self, msg):
+    def __init__(self,msg,ctx,ips,user,password):
```

Comment:
nit: Generally we follow Python PEP3 guidelines for our code. That means all the method definitions and calls should have spaces between args.
```
def func1(a, b, c, d):
      func2(x, y, z)
```
Can we update the relevant places with this change to keep it consistent?

Reply:
Done.